arviz/plots/backends/matplotlib/distplot.py

```diff
@@ -43,11 +44,16 @@ def plot_dist(
         )
         backend_kwargs = None
     if ax is None:
-        ax = plt.gca()
+        ax = plt.gca(polar=is_circular)
```

Comment:
Is using `gca` instead of a new axes intended or a forgotten TODO from when bokeh was introduced? cc @ahartikainen @canyon289 @aloctavodia

Reply:
I think this is intended

Reply:
Will `gca` grab possibly old axis here? Have we created a new figure somewhere?

Reply:
if present, `plt.gca` will grab old axes yes

Reply:
I believe this is considered a "best practice" with matplotlib, and a pattern they endorse.